backend/LexBoxApi/LexBoxApi.csproj

```diff
@@ -11,24 +11,25 @@
 
     <ItemGroup>
         <PackageReference Include="DataAnnotatedModelValidations" Version="4.1.0" />
-        <PackageReference Include="HotChocolate.Types.Analyzers" Version="13.5.1">
+        <PackageReference Include="HotChocolate.Analyzers" Version="13.6.0-preview.21" />
+        <PackageReference Include="HotChocolate.Types.Analyzers" Version="13.6.0-preview.21">
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
-        <PackageReference Include="HotChocolate.AspNetCore" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.Data.EntityFramework" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.Diagnostics" Version="13.5.1" />
+        <PackageReference Include="HotChocolate.AspNetCore" Version="13.6.0-preview.21" />
+        <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="13.6.0-preview.21" />
+        <PackageReference Include="HotChocolate.Data.EntityFramework" Version="13.6.0-preview.21" />
+        <PackageReference Include="HotChocolate.Diagnostics" Version="13.6.0-preview.21" />
         <PackageReference Include="Humanizer.Core" Version="2.14.1" />
         <PackageReference Include="MailKit" Version="4.0.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="7.0.3" />
-        <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.2" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.2">
+        <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0-rc.1.23421.29" />
```

Comment:
These Microsoft packages can now be version 8.0.0 rather than release candidates.